drivers/adc/gecko_adc.c

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright (c) 2022 T-Mobile USA, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT silabs_adc_gecko
+
+#include <string.h>
+#include <zephyr/drivers/adc/gecko_adc/gecko_adc.h>
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/devicetree.h>
+
+#include "em_cmu.h"
+#include "em_gpio.h"
```

Comment:
Since these header files aren't colocated with this module, the non-local syntax form should be employed:
```
 #include <em_cmu.h>
 #include <em_gpio.h>
```